### PR TITLE
Minimize README contents

### PR DESCRIPTION
**Pull request by megnvidia**
# What does this PR do ?

* quick start is in doc. Putting link in readme rather than duplicate content
* created install.md moved and formatted readme/install to that file. Added TOC link to new file.
* stole About content from in process quick start, left news in(it is in the sglang), moved our quick start link after the news
* added overview.md file, moved readme content (megatron-LM -->ecosystem libraries) to overview, merged a bit of my draft overview content from before. Overview needs some technical massage, I have some points wrong in my descriptions.
* adding releasenotes.md and manipulating the TOC for compatibility with doc team structure template (SWDOCS&title=Peer+Review+Checklist)
* removing training section because content duplicates better placed: https://docs.nvidia.com/megatron-core/developer-guide/latest/user-guide/data-preparation.html and https://docs.nvidia.com/megatron-core/developer-guide/latest/user-guide/training-examples.html
* removing Parallelism Strategies section because content duplicates better placed: https://docs.nvidia.com/megatron-core/developer-guide/latest/user-guide/parallelism-guide.html
* moved roadmaps to release notes
* changed title of community and support to Resources to align with swdocs structure template

## Contribution process

```mermaid
flowchart LR
    A[Pre-checks] --> B[PR Tests]
    subgraph Code Review/Approval
        C1[Expert Review] --> C2[Final Review]
    end
    B --> C1
    C2 --> D[Merge]
```

### Pre-checks

- [ ] I want this PR in a versioned release and have added the appropriate Milestone (e.g., `Core 0.8`)
- [ ] I have added relevant unit tests
- [ ] I have added relevant functional tests
- [ ] I have added proper typing to my code [Typing guidelines](https://docs.python.org/3/library/typing.html)
- [ ] I have added relevant documentation
- [ ] I have run the [autoformatter.sh](https://github.com/NVIDIA/Megatron-LM/blob/main/tools/autoformat.sh) on my PR

### Code review

The following process is enforced via the CODEOWNERS file for changes into `megatron/core`. For changes outside of `megatron/core`, it is up to the PR author whether or not to tag the Final Reviewer team.

<details>
<summary>For MRs into `main` branch</summary>

Feel free to message or comment the @mcore-oncall to help accelerate your merge into main. The less complex your PR is, the faster it will be approved and merged!

#### (Step 2): Collect the expert reviewers reviews

1. Attach the `Expert Review` label when your PR is ready for review.
2. GitHub auto-assigns expert reviewers based on your changes. They will get notified and pick up your PR soon.

:warning: Only proceed to the next step once all reviewers have approved, merge-conflict are resolved and the CI is passing.  
Final Review might get declined if these requirements are not fulfilled.

#### (Step 3): Final Review

1. Add `Final Review` label
2. GitHub auto-assigns final reviewers based on your changes. They will get notified and pick up your PR soon.

#### (Optional Step 4): Cherry-pick into release branch

If this PR also needs to be merged into `core_r*` release branches, after this PR has been merged, select `Cherry-pick` to open a new PR into the release branch.

</details>

<details>
<summary>For MRs into `dev` branch</summary>
The proposed review process for `dev` branch is under active discussion.

MRs are mergable after one approval by either `eharper@nvidia.com` or `zijiey@nvidia.com`.
</details>

### Merging your PR

Any member of [core-adlr](https://github.com/orgs/teams/NVIDIA/core-adlr) and [`core-nemo`](https://github.com/orgs/teams/NVIDIA/core-nemo) will be able to merge your PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,19 +11,17 @@ Megatron-LM & Megatron Core
...
+You can install this library using pip or use it within the Megatron-LM GitHub repository.
 
-# 2. Clone repository for examples
-git clone https://github.com/NVIDIA/Megatron-LM.git
-cd Megatron-LM
-pip install --no-build-isolation .[mlm,dev]
-```
+**Megatron-LM**: GitHub repository that includes Megatron-Core, as well as example training loops utilizing Megatron-Core.
+
+**Megatron-Bridge**: a separate repository that is ideal for users who wish to train a Hugging Face model.
```
Comment (Phlip79) [inline README.md]: Please add link: https://github.com/NVIDIA-NeMo/Megatron-Bridge.

File: README.md [hunk 2]
```diff
@@ -11,19 +11,17 @@ Megatron-LM & Megatron Core
...
+It serves as a powerful bridge, conversion, and verification layer between Hugging Face and Megatron Core.
 
-**→ [Complete Installation Guide](#installation)** - Docker, pip variants (dev,lts,etc.), and system requirements
+**NeMo RL**: library that streamlines and scales reinforcement learning, that leverages Megatron-Core.
```
Comment (Phlip79) [inline README.md]: Please add link: https://github.com/NVIDIA-NeMo/RL.

File: docs/get-started/overview.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Overview
+
+Megatron-Core and Megatron-LM are open-source tools that are typically used together to train LLMs at scale across GPUs. Megatron-Core expands the capability of Megatron-LM. Megatron Bridge connects Megatron-Core and Megatron-LM to other popular training models, such as Hugging Face.
+
+
+## Megatron-LM:
```
Comment (Phlip79) [inline docs/get-started/overview.md]:
```suggestion
## Megatron-LM
```

File: docs/get-started/overview.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+- **Learning distributed training** concepts and best practices
+- **Quick experimentation** with proven model configurations
+
+**What you get:**
+
+- Pre-configured training scripts for GPT, LLaMA, DeepSeek, Qwen, and more.
+- End-to-end examples from data prep to evaluation
+- Research-focused tools and utilities
+
+## Megatron Core
```
Comment (Phlip79) [inline docs/get-started/overview.md:5]: Please put the `Megatron Core` section before `Megatron-LM`

File: docs/get-started/overview.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+NVIDIA Megatron Core is a library of essential building blocks for highly efficient large-scale generative AI training. It can be used to train models with unparalleled speed at scale across thousands of GPUs. It provides an extensive set of tools for multimodal and speech AI. It expands Megatron LM capabilities.
+
+Megatron-Core contains GPU-optimized techniques featuring advanced parallelism strategies, optimizations like FP8 training, and support for the latest LLM, MoE, and multimodal architectures. It abstracts these techniques into composable and modular APIs.
+
+Megatron-Core is compatible with all NVIDIA Tensor Core GPUs and popular LLM architectures such as GPT, BERT, T5, and RETRO.
+
+[![Megatron-Core Documentation](https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat)](https://docs.nvidia.com/Megatron-Core/developer-guide/latest/index.html)
```
Comment (Phlip79) [inline docs/get-started/overview.md]: Should this be removed?

File: docs/get-started/install.md
Comment (Phlip79) [inline docs/get-started/install.md:1]: Can you fix section titles to make them use ## instead of #. Also, can we remove colons from titles?

Comment (megnvidia) [reply docs/get-started/install.md:1]: happy to

File: docs/get-started/overview.md (hunk 5 above)
Comment (megnvidia) [reply docs/get-started/overview.md]: yes

File: README.md
Comment (Phlip79) [inline README.md]:
```suggestion
For our latest performance benchmarking results, please refer to [NVIDIA NeMo Framework Performance Summary](https://docs.nvidia.com/nemo/megatron-bridge/latest/performance-summary.html).
```

File: README.md [hunk 6]
```diff
@@ -11,19 +11,19 @@ Megatron-LM & Megatron Core
 
 <div align="left">
 
-## ⚡ Quick Start
```
Comment (sbhavani) [inline README.md:14]: please leave the quick start at the top for easy installation

File: README.md [hunk 7]
```diff
@@ -11,19 +11,19 @@ Megatron-LM & Megatron Core
...
+## About
 
-```bash
-# 1. Install Megatron Core with required dependencies
-pip install --no-build-isolation megatron-core[mlm,dev]
+**Megatron-Core (MCore)**: the core, composable library with GPU-optimized building blocks for custom training frameworks
```
Comment (sbhavani) [inline README.md]: Megatron Core: Composable library with GPU-optimized building blocks for custom training frameworks.

File: README.md [hunk 8]
```diff
@@ -11,19 +11,19 @@ Megatron-LM & Megatron Core
...
+You can install this library using pip or use it within the Megatron-LM GitHub repository.
 
-# 2. Clone repository for examples
-git clone https://github.com/NVIDIA/Megatron-LM.git
-cd Megatron-LM
-pip install --no-build-isolation .[mlm,dev]
-```
+**Megatron-LM**: GitHub repository that includes Megatron-Core, as well as example training loops utilizing Megatron-Core.
```
Comment (sbhavani) [inline README.md]: Megatron-LM: Reference implementation that includes end-to-end examples utilizing Megatron Core.

File: README.md [hunk 9]
```diff
@@ -11,19 +11,19 @@ Megatron-LM & Megatron Core
...
+
+**Megatron-Bridge**: a separate repository that is ideal for users who wish to train a Hugging Face model.
```
Comment (sbhavani) [inline README.md]: Megatron Bridge: Training library with bidirectional Hugging Face ↔ Megatron checkpoint conversion, flexible training loops, and example model training recipes.

File: README.md [hunk 10]
```diff
@@ -11,19 +11,19 @@ Megatron-LM & Megatron Core
...
+It serves as a powerful bridge, conversion, and verification layer between Hugging Face and Megatron Core.
+For more information, refer to [Megatron Bridge](https://github.com/NVIDIA-NeMo/Megatron-Bridge).
 
-**→ [Complete Installation Guide](#installation)** - Docker, pip variants (dev,lts,etc.), and system requirements
+**NeMo RL**: library that streamlines and scales reinforcement learning, that leverages Megatron-Core.
```
Comment (sbhavani) [inline README.md]: I think we can leave this one out or keep an ecosystem section with all the libraries that build on Megatron Core

File: README.md (hunk 6 above)
Comment (megnvidia) [reply README.md:14]: requirements are super short. I really do recommend having requirements first.
